pkg/apis/kubermatic/v1/policy_template.go

```diff
@@ -83,8 +66,12 @@ type PolicyTemplateSpec struct {
 	// A PolicyBinding referencing it cannot disable it
 	Enforced bool `json:"enforced"`
 
-	// KyvernoSpec is the Kyverno specification
-	kyvernov1.Spec `json:",inline"`
+	// Policy is the policy we want to apply on the cluster
```

Comment:
Need some documentation/link etc here to inform users what the "Policy" or its spec should look like.